<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PureMediaServer</class>
 <widget class="QMainWindow" name="PureMediaServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pure Media Server</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>91</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Pure Media Server 0.0.5
Copyright (C) 2012 Santiago Nore√±a  
puremediaserver@gmail.com
This program comes with ABSOLUTELY NO WARRANTY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Media</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectPDButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect To PD</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="layer1Check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="layer2Check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="layer3Check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="layer4Check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="layer5Check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="layer6Check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="layer7Check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="layer8Check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="layer1Add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="layer2Add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="layer3Add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="layer4Add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="layer5Add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="layer6Add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="layer7Add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="layer8Add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="readDMX">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>290</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Read DMX</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="universe">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>340</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="winpositionx">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>320</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>OLA Universe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Window Position</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="winpositiony">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="winsizey">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>220</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Window Size</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="winsizex">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QLabel" name="layer1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="layer2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="layer3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="layer4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="layer5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="layer6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer 6</string>
    </property>
   </widget>
   <widget class="QLabel" name="layer7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer 7</string>
    </property>
   </widget>
   <widget class="QLabel" name="layer8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer 8</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="window">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Path</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ipAddress1">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>254</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="ipAddress2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>254</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="ipAddress3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>390</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>254</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="ipAddress4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>254</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_conf"/>
    <addaction name="actionSave_conf"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen_conf">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open conf</string>
   </property>
  </action>
  <action name="actionSave_conf">
   <property name="text">
    <string>Save conf</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
